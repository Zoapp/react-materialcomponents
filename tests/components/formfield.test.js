import React from "react";
import { shallow } from "enzyme";
import FormField from "libs/components/formField";

describe("components/FormField", () => {
  const wrapper = shallow(
    <FormField>
      <div className="child" />
    </FormField>,
  );

  it("can have children", () => {
    expect(wrapper.contains(<div className="child" />)).toEqual(true);
  });
});
