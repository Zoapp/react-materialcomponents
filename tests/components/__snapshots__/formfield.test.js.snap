// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/FormField can have children 1`] = `
<div
  className="mdc-form-field"
>
  Children is here!
</div>
`;
